### PR TITLE
FUSETOOLS-3465 - provide github action for basic PR job

### PR DESCRIPTION
**Pull request by apupier**
to see the github actions working, need to look to my fork https://github.com/apupier/jbosstools-fuse/pull/12

### CODE REVIEW

**Review by bfitzpat — APPROVED** [review]

**Comment by apupier** [comment]
Ci status not updated on PSI Jenkins but they are in success:
- https://studio-jenkins-csb-codeready.cloud.paas.psi.redhat.com/job/Studio/job/Engineering/job/pr_checks/job/pullrequest-fuse/20/
- https://studio-jenkins-csb-codeready.cloud.paas.psi.redhat.com/job/Studio/job/Engineering/job/pr_checks/job/pullrequest-fuse-templates/20/
(and corresponding job will be removed anyway after merge)

**Comment by apupier** [comment]
do not merge because we are in code freeze

**Review by lhein — APPROVED** [review]

**Pull request merged**
